pkg/controller/upgradeconfig/cluster_upgrader.go

```diff
@@ -568,7 +570,7 @@ func performClusterHealthCheck(c client.Client) (bool, error) {
 
 	secret := &corev1.Secret{}
 
-	err = c.Get(context.TODO(), types.NamespacedName{"openshift-monitoring", tokenSecret}, secret)
+	err = c.Get(context.TODO(), types.NamespacedName{Namespace: "openshift-monitoring", Name: tokenSecret}, secret)
 	if err != nil {
 		return false, err
```

Comment:
As per my comment on line 594.

```suggestion
		return false, fmt.Errorf("Unable to fetch secret %s: %s", tokenSecret, err)
```

Reply:
Thanks a lot for the review Matt ! Have submitted changes..